lib/Settings.js

```diff
@@ -36,15 +37,25 @@ Settings.loadRepositoriesSettings = function (configPath) {
     }
     repositoryNames = Object.keys(configJson);
     if (repositoryNames.length === 0) {
-        return Promise.reject('`repositories` be non-empty');
+        return Promise.reject('`repositories` must be non-empty');
     }
     for (var i = 0; i < repositoryNames.length; i++) {
-        name = repositoryNames[i];
-        if (!defined(configJson[name]['gitHubToken'])) {
-            return Promise.reject('repository ' + name + ' must have a `gitHubToken`');
+        full_name = repositoryNames[i];
+        if (!/\//.test(full_name)) {
+            return Promise.reject('repository ' + full_name + ' must be in the form {user}/{repository}');
         }
+        // Modify repository names to be accessed by their short names
+        short_name = full_name.split('/')[1];
+        var tmp = configJson[full_name];
+        configJson[short_name] = tmp;
+        delete configJson[full_name];
+        repositoryNames[i] = short_name;
 
```

Comment:
This block is pretty ugly. Any better way to do it?

This ugliness could be solved if I had more control of how `express-github-webhook` passes the repository name. I think, as we talked about offline, we do not need the dependency. 

That's a task for another day, though.